instrumentation/spring/spring-boot-autoconfigure/README.md

```diff
@@ -412,10 +412,10 @@ This is the preferred way to enable/disable exporters and takes precedence over
 
 ##### Resource Properties
 
-| Feature  | Property                                                            | Default Value          |
-| -------- |---------------------------------------------------------------------| ---------------------- |
-| Resource | otel.springboot.resource.enabled                                    | `true`                 |
-|          | otel.resource.attributes (old: otel.springboot.resource.attributes) | `empty map`            |
+| Feature  | Property                         | Default Value |
+|----------|----------------------------------|---------------|
+| Resource | otel.springboot.resource.enabled | `true`        |
+|          | otel.resource.attributes         | `empty map`   |
```

Comment:
can you update the PR title if it includes removing old property names, since we'll want to include that info in the migrating section at the top of the change log?

Reply:
it's only removing the documentation - not the functionality

Reply:
is it related to this PR somehow? if not, it really helps to avoid confusion by not mixing unrelated things in a single PR

Reply:
the idea was to deprecate this property - but I've reverted the change